lib/improver/tests/ensemble_calibration/ensemble_calibration/test_ApplyCoefficientsForEnsembleCalibration.py

```diff
@@ -44,13 +44,56 @@
     ApplyCoefficientsFromEnsembleCalibration as Plugin)
 from improver.ensemble_calibration.ensemble_calibration import (
     EstimateCoefficientsForEnsembleCalibration)
+from improver.tests.ensemble_calibration.ensemble_calibration. \
+    helper_functions import SetupCubes, EnsembleCalibrationAssertions
 from improver.tests.ensemble_calibration.ensemble_calibration.\
     test_EstimateCoefficientsForEnsembleCalibration import (
-        create_coefficients_cube)
+        create_coefficients_cube, SetupExpectedCoefficients)
 from improver.tests.set_up_test_cubes import set_up_variable_cube
 from improver.utilities.warnings_handler import ManageWarnings
 
 
+class SetupCoefficientsCubes(SetupCubes, SetupExpectedCoefficients):
+
+    """Set up coefficients cubes for testing."""
+
+    @ManageWarnings(
+        ignored_messages=["Collapsing a non-contiguous coordinate.",
+                          "invalid escape sequence",
+                          "can't resolve package from",
+                          "The statsmodels can not be imported"],
+        warning_types=[UserWarning, DeprecationWarning, ImportWarning,
+                       ImportWarning])
+    def setUp(self):
+        """Set up coefficients cubes for when either the ensemble mean or the
+        ensemble realizations have been used as the predictor. The coefficients
+        have been constructed from the same underlying set of ensemble
+        realizations, so application of these coefficients would be expected
+        to give similar results."""
+        super().setUp()
+        # Set up a coefficients cube when using the ensemble mean as the
```

Comment:
It might be worth explicitly saying which class of tests the expected data is taken from to form the input data of these tests. I can imagine someone may get confused by the web of tests depending on other tests if we aren't careful. It probably wouldn't hurt to say where the expected arrays are set up that they are used as input into these tests.

Reply:
I've added a couple of sentences to the docstring for the class. Is anything else required?